src/components/ChatView.vue

```diff
@@ -23,7 +23,7 @@
 		@dragover.prevent="handleDragOver"
 		@dragleave.prevent="handleDragLeave"
 		@drop.prevent="handleDropFiles">
-		<GuestWelcomeWindow v-if="isGuestWithoutDisplayName" :token="token" />
+		<GuestWelcomeWindow v-if="showGuestNameWindow" :token="token" />
```

Comment:
```suggestion
		<GuestWelcomeWindow v-if="showGuestWelcomeWindow" :token="token" />
```

I'd name this computed (and prop) the same way we named a component, for consistensy